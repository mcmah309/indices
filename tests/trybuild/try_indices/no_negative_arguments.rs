use indices::try_indices;

fn main() {
    let mut data = [5,4,3,2,1];
    let slice = data.as_mut_slice();
    let result = try_indices!(slice, -1, 1);
}