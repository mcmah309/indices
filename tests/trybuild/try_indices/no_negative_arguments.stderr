error[E0600]: cannot apply unary operator `-` to type `usize`
 --> tests/trybuild/try_indices/no_negative_arguments.rs:6:38
  |
6 |     let result = try_indices!(slice, -1, 1);
  |                                      ^^
  |                                      |
  |                                      cannot apply unary operator `-`
  |                                      help: you may have meant the maximum value of `usize`: `usize::MAX`
  |
  = note: unsigned values cannot be negated
